### DISCUSSION TITLE
Indication of missing required fields for new discussion isn't great

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44235] [Mobile] [answered]
GitHub Mobile beta

Create a discussion
Fill in the content first because there are a bunch of things in the clipboard and Android's clipboard might decide to empty itself otherwise...

Once you've written enough content, the title field will have scrolled off screen (it's easier if you insert a couple of screenshots):

![Screenshot_20230113-102443.png](https://user-images.githubusercontent.com/2119212/212356256-96737ba5-ff76-4df6-9d0b-690b88c30e02.png)

![Screenshot_20230113-102519.png](https://user-images.githubusercontent.com/2119212/212356324-3760555b-6557-4174-89a2-5accde0624e9.png)

Tap the send button.

Nothing happens. 

Technically, there's a distinction between the blue send and the gray send, but that isn't much:
![Screenshot_20230113-102639.png](https://user-images.githubusercontent.com/2119212/212356622-70a5b186-0944-45cc-a415-f5038177220c.png)

In poor lighting it isn't noticable.
It'd be nicer if something showed below the send button saying that the description isn't filled out when one taps the button.

If there are multiple missing mandatory fields, including a red count of them next to the indicator would help immensely. 

For reference, here are the Slack for Android buttons for send a message when there is or isn't a message to send:
![Screenshot_20230113-103751](https://user-images.githubusercontent.com/2119212/212359143-b98e314f-99ce-436c-b362-6ea25ad2ac3f.png)
![Screenshot_20230113-103802](https://user-images.githubusercontent.com/2119212/212359159-e859fbaa-b144-4636-91a4-e2070d1502f4.png)
It's really obvious that they're different states.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for the feedback. I passed this along to the team for consideration!
